core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -263,6 +263,21 @@
       </div>
     </script>
 
+    <script type="text/ng-template" id="modals/welcomeExploration">
```

Comment:
It looks pretty good a bit smaller. I changed it to only take up 70% of the screen. Had to add some more CSS to vertically align. Also, it is now just fading in instead of sliding down. May have to do with removing it from oppia CSS